### DISCUSSION TITLE
devel -> master cadence

### DISCUSSION THREAD

**Discussion by haz** [aig-upf/tarski#111] [General]
How often can we expect to see the `devel` channel merged with `master` and deployed on PyPi? We have some CI/CD that fails on the publically available `tarski`, mainly because we lean on some of the changes in the `devel` branch.

**Comment by gfrances** [comment]
We don't really have a fixed schedule for pushing to pypi.
Could you guys share some details of your CI config? It could be possible to simply `pip install` from the github repo **at a concrete revision ID**, which might be all you need?

**Reply by haz** [reply]
Ah, interesting. Currently we bring it in via requirements:

```if [ -f requirements.txt ]; then pip install -r requirements.txt; fi```

And this is what's listed:

```tarski>=0.7.0```

So something like [[this](https://stackoverflow.com/a/35998253/1026323)] should work until we've stabilized what we need from tarski, and that's reflected in `master`/PyPi?

CC'ing @e-cal

**Reply by gfrances** [reply]
Yes, for instance [this](https://github.com/rleap-project/d2l/blob/main/setup.py#L64) is how I pin down a specific Git-commit-version of Tarski in one of my projects using it. That one is in a `setup.py`, but should work similarly in the `requirements.txt` file, as in the link you provide.
Let me know if that works, if not, we probably can find some other workaround.
